src/components/Table/DragHandle.tsx

```diff
@@ -0,0 +1,39 @@
+import clsx from "clsx";
+import { Icon } from "components";
+import styled from "styled-components";
+
+interface HandleWrapperProps {
+	isDragging?: boolean;
+}
+
+const HandleWrapper = styled.div<HandleWrapperProps>`
+  height: 16px;
+  width: 16px;
+  vertical-align: center;
+  display: inline-block;
+  margin-right: 0.5rem;
+  cursor: ${({ isDragging }) => (isDragging ? "grabbing" : "grab")};
+`;
+
+interface DragHandleProps {
+	isDragging?: boolean;
+	// biome-ignore lint/suspicious/noExplicitAny: <explanation>
+	[key: string]: any; // To allow spreading other props
```

Comment:
nit: combine comments
```suggestion
	// biome-ignore lint/suspicious/noExplicitAny: To allow spreading other props
	[key: string]: any;
```